manifests/m/Microsoft/BotFrameworkComposer/2.0.0/Microsoft.BotFrameworkComposer.yaml

```
@@ -0,0 +1,25 @@
+PackageIdentifier: Microsoft.BotFrameworkComposer
+PackageVersion: 2.0.0
+PackageName: Bot Framework Composer
+Publisher: Microsoft Corporation
+Author: Microsoft
+License: Copyright (c) Microsoft Corporation
+LicenseUrl: https://github.com/microsoft/BotFramework-Composer/blob/master/LICENSE.md
+Tags:
+- bot
+- framework
+- composer
+ShortDescription: Microsoft.BotFrameworkComposer
+Description: Bot Framework Composer is an integrated development tool for developers and multi-disciplinary teams to build bots and conversational experiences with the Microsoft Bot Framework. Within this tool, you'll find everything you need to build a sophisticated conversational experience.
+PackageUrl: https://github.com/microsoft/BotFramework-Composer
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/microsoft/BotFramework-Composer/releases/download/v2.0.0/BotFramework-Composer-2.0.0-windows-setup.exe
+  InstallerSha256: 93371f6ed1536c49328eab9d3a7252c31eaa0aeffbae2323437789eb4fa58e8d
+  InstallerType: exe
+  InstallerSwitches:
+    Silent: /S
+    SilentWithProgress: /S
```

```suggestion
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/microsoft/BotFramework-Composer/releases/download/v2.0.0/BotFramework-Composer-2.0.0-windows-setup.exe
  InstallerSha256: 93371f6ed1536c49328eab9d3a7252c31eaa0aeffbae2323437789eb4fa58e8d
  InstallerType: nullsoft
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/microsoft/BotFramework-Composer/releases/download/v2.0.0/BotFramework-Composer-2.0.0-windows-setup.exe
  InstallerSha256: 93371f6ed1536c49328eab9d3a7252c31eaa0aeffbae2323437789eb4fa58e8d
  InstallerType: nullsoft
  InstallerSwitches:
    Custom: /ALLUSERS
```
Some cleanup and compatibility